app/views/layouts/application.html.slim

```diff
@@ -12,7 +12,17 @@ html dir="ltr" lang="en-GB" class="govuk-template"
     = tag.meta name: "viewport", content: "width=device-width, initial-scale=1"
     = tag.meta property: "og:image", content: asset_path("images/govuk-opengraph-image.png")
     = tag.meta name: "theme-color", content: "#0b0c0c"
-    = tag.meta name: "robots", content: "noindex"
+    ruby:
+      index_prod = FeatureFlags.enabled?(:index_production, without_session_data: true) ? "" : "noindex"
+    = tag.meta name: "robots", content: index_prod.to_s
+
+    ruby:
+      canoncical_link = if ENV["PRIMARY_HOST"].present?
+                          "https://check-your-client-qualifies-for-legal-aid.service.gov.uk#{url_for(only_path: true, protocol: 'https')}"
+                        end
+      use_canionical = ENV["PRIMARY_HOST"].present? ? "canonical" : nil
+
+    link[rel="#{use_canionical}" href='<%= #{canoncical_link} %>']
```

Comment:
That `<%=` is ERB syntax that won't be correctly rendered here. You'll end up with HTML that looks like:

```html
 <link rel="canonical" href="<%= https://check-your-client-qualifies-for-legal-aid.service.gov.uk/ %">
 ```
 
 And that's not a valid href value.